pkg/service/repair/progress.go

```diff
@@ -261,12 +262,12 @@ func (pm *dbProgressManager) emptyState() bool {
 	return true
 }
 
-func (pm *dbProgressManager) GetCompletedRanges(keyspace, table string) []scyllaclient.TokenRange {
+func (pm *dbProgressManager) GetCompletedRanges(keyspace, table string) (doneRanges []scyllaclient.TokenRange, allRangesCnt int) {
```

Comment:
What you need named return parameters for ?
They are not used.

Reply:
It was a tip from linter. It suggests to name returned values when their type doesn't make it obvious.